api/PasswordCredential.json

```diff
@@ -149,13 +149,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential/iconURL",
           "support": {
             "chrome": {
-              "version_added": "52"
+              "version_added": "51"
```

Comment:
https://storage.googleapis.com/chromium-find-releases-static/be6.html#be6cc81fc74f63c8e134525bbf15ea966c8403d5 says this was in 52, but it was cherry-picked for 51:
https://bugs.chromium.org/p/chromium/issues/detail?id=602980#c2